# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/cocoa/MediaHardwareKeysEventSourceMacMediaCenter.h
# Commit: 714b2fb21e23
# Full Hash: 714b2fb21e2341e8e2bf1be1271ca0ed5e526443
# Author: alwu <alwu@mozilla.com>
# Date: 2020-06-09 09:21:34
# Regressor Bug: 1640998
# File Overlap Count: 1
# Description:
#   Bug 1640998 - part5 : set supported keys to the event source r=chunmin
#   
#   This patch will
#   - add a method `SetSupportedMediaKeys()` on `MediaControlKeysEventSource`
#   - set main controller's supported key to the event source
# ==============================================================================

diff -r a48b6554ba9c -r 714b2fb21e23 widget/cocoa/MediaHardwareKeysEventSourceMacMediaCenter.h
--- a/widget/cocoa/MediaHardwareKeysEventSourceMacMediaCenter.h	Mon Jun 08 22:16:46 2020 +0000
+++ b/widget/cocoa/MediaHardwareKeysEventSourceMacMediaCenter.h	Tue Jun 09 02:23:35 2020 +0000
@@ -35,6 +35,8 @@
   void Close() override;
   bool IsOpened() const override;
   void SetPlaybackState(dom::MediaSessionPlaybackState aState) override;
+  // Currently we don't support showing supported keys on the touch bar.
+  void SetSupportedMediaKeys(const MediaKeysArray& aSupportedKeys) override {}
 
  private:
   ~MediaHardwareKeysEventSourceMacMediaCenter();